src/integrationTest/resources/tests/transcriptions/transcription/expectedResponse.json

```diff
@@ -1,12 +1,12 @@
 {
   "case_number": "1",
-  "courthouse_id" : $COURTHOUSE_ID,
+  "courthouse_id": $COURTHOUSE_ID,
   "courthouse": "some-courthouse",
```

Comment:
Should this be uppercase?

Reply:
no, this is showing the display_name, so wont be upper-case, as per https://tools.hmcts.net/jira/browse/DMP-3651